src/main/scala/nl/knaw/dans/easy/multideposit/package.scala

```diff
@@ -254,6 +276,10 @@ package object multideposit {
   }
 
   implicit class DatasetExtensions(val dataset: Dataset) extends AnyVal {
+    def getRowNumber: Int = {
+      dataset("ROW").head.toInt // first occurrence of dataset, assuming it is not empty
```

Comment:
What do you mean "_If `dataset("ROW")` is a stream_"? `Dataset` has type `Map[String, List[String]]`, so `dataset("ROW")` has type `List[String]`. Also, `head` takes the first element from a list, if the list is non-empty.